static/src/components/performancestats/index.vue

```diff
@@ -95,6 +111,9 @@ export default {
       type: Object,
       default: () => {}
     },
+    csrfToken: {
```

Comment:
I'm not sure if the test for this component will break since there is a  new prop to the component. Also maybe we can make  it  required: true or added a default value

Reply:
I'm adding a `read_only` option so the CSRF will not need to be required

Reply:
ok, cool I think the lint may complain if the is no default but otherwise looks good to me

Reply:
in that case it will get a default value of `undefined` ¯\\\_(ツ)\_/¯